docs/sources/writing-guide/about-documentation-design/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: About our documentation design
+menuTitle: Documentation design
+description: Learn about Grafana's documentation docs pages
+aliases:
+  - /docs/writers-toolkit/latest/writing-guide/about-documentation-design/
+weight: 100
+keywords:
+  - Grafana
+  - documentation
+  - page design
+---
+
+# About our documentation design
+
+In late 2022, the documentation website was significantly modified to a more modern design. The goal of the redesign was to make our technical documentation more accessible, modern, and scalable as we grow.  
```

Comment:
@GrafanaWriter , Bits that include dates and words like 'new' and 'now' become stale quickly. Maybe start at "Our docs page take sadvantage of our static site generator, Hugo. As a result, several page elements are automatically managed during the publication ..." - and remove everything before that content?

If you are good with that approach, I can update the PR. Thanks!

Reply:
yes, please.